src/haz3lweb/Keyboard.re

```diff
@@ -3,6 +3,219 @@ open Haz3lcore;
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Toggle Dynamics",
+    UpdateAction.Set(Dynamics),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Elaboration",
+    UpdateAction.Set(Elaborate),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Function Bodies",
+    UpdateAction.Set(Evaluation(ShowFnBodies)),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Case Clauses",
+    UpdateAction.Set(Evaluation(ShowCaseClauses)),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show fixpoints",
+    UpdateAction.Set(Evaluation(ShowFixpoints)),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Casts",
+    UpdateAction.Set(Evaluation(ShowCasts)),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Lookup Steps",
+    UpdateAction.Set(Evaluation(ShowLookups)),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Stepper Filters",
+    UpdateAction.Set(Evaluation(ShowFilters)),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Hidden Steps",
+    UpdateAction.Set(Evaluation(ShowHiddenSteps)),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Docs Sidebar",
+    UpdateAction.Set(ExplainThis(ToggleShow)),
+  ),
+  mk_shortcut(
+    ~section="Settings",
+    ~mdIcon="tune",
+    "Toggle Show Docs Feedback",
+    UpdateAction.Set(ExplainThis(ToggleShowFeedback)),
+  ),
+  mk_shortcut(
+    ~hotkey=meta(sys) ++ "+/",
+    ~mdIcon="assistant",
+    "TyDi Assistant",
+    Assistant(Prompt(TyDi)) // I haven't figured out how to trigger this in the editor
+  ),
+  mk_shortcut(
+    ~mdIcon="download",
+    ~section="Export",
+    "Export Scratch Slide",
+    Export(ExportScratchSlide),
+  ),
+  mk_shortcut(
+    ~mdIcon="download",
+    ~section="Export",
+    "Export All Persistent Data",
+    Export(ExportPersistentData),
+  ),
+  mk_shortcut(
+    ~mdIcon="download",
+    ~section="Export",
+    "Export Exercise Module",
+    Export(ExerciseModule) // TODO Would we rather skip contextual stuff for now or include it and have it fail
+  ),
+  mk_shortcut(
+    ~mdIcon="download",
+    ~section="Export",
+    "Export Submission",
+    Export(Submission) // TODO Would we rather skip contextual stuff for now or include it and have it fail
+  ),
+  mk_shortcut(
+    ~mdIcon="download",
+    ~section="Export",
+    "Export Transitionary Exercise Module",
+    Export(TransitionaryExerciseModule) // TODO Would we rather skip contextual stuff for now or include it and have it fail
+  ),
+  mk_shortcut(
+    ~mdIcon="download",
+    ~section="Export",
+    "Export Grading Exercise Module",
+    Export(GradingExerciseModule) // TODO Would we rather skip contextual stuff for now or include it and have it fail
```

Comment:
These are the contextual entries I'm just showing for now that will fail for now.

Reply:
Remove instructor ones if built for instructor mode